core/src/types/operator/operator.rs

```diff
@@ -256,6 +256,25 @@ impl Operator {
     /// # }
     /// ```
     ///
+    /// ## `version`
+    ///
+    /// Set `version` for this `stat` request.
+    ///
+    /// This feature can be used to retrieve the file metadata that matches a specified version.
```

Comment:
Hi, the way about discussing `version` is a bit confusing to me. Unlike `ETag`, `Version` is not used for matching. Instead, we request the specified version directly. We will return `NotFound` if the requested version does not exist for the given path.

Reply:
how about 
```
    /// This feature can be used to retrieve the path's metadata of a specified version.
    ///
    /// If the version doesn't exist, an error with kind [`ErrorKind::NotFound`] will be returned.
```